website/src/components/KnowTheCommunityCarousel/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import KnowTheCommunityCard from '../KnowTheCommunityCard/index';
+import styles from './index.module.css';
+import features from '../../../websiteData/features';
+import { PhCaretLeftFill, PhCaretRightFill } from '../../assets/icon';
+
+const index = () => {
+  const [curSlide, setCurSlide] = useState(0);
+
+  const onLeftClick = () => {
+    if (curSlide === 0) {
+      setCurSlide(-8);
```

Comment:
Hardcoding this value leaves extra space at the end. Can you brainstorm on how you can make this dynamic?

Reply:
We can make that dynamic by changing that hard-coded value depending on the screen size

Reply:
Or maybe you can set the number of clicks depending on how many screens are being rendered. Either works.